build.gradle

```diff
@@ -97,6 +97,7 @@ subprojects {
       // the "value" -quiet is added, separated by a space. This is ok since the javadoc
       // command already adds -quiet, so we are just duplicating it
       // see https://discuss.gradle.org/t/add-custom-javadoc-option-that-does-not-take-an-argument/5959
+      javadoc.options.encoding='UTF8'
```

Comment:
That should be a separate PR. On windows, not having that option triggers some encoding exceptions (since Windows replies on its own cp1252 scheme). It can be moved to `BuildPlugin` just as well.